server/protocols/jmap/src/test/java/org/apache/james/jmap/utils/JsoupHtmlTextExtractorTest.java

```diff
@@ -53,14 +52,14 @@ public void toPlainTextShouldRemoveSimpleHtmlTag() {
     @Test
     public void toPlainTextShouldReplaceSkipLine() {
         String html = "<p>This is an<br/>HTML text !</p>";
-        String expectedPlainText = "This is an\nHTML text !\n";
+        String expectedPlainText = "This is an HTML text !";
```

Comment:
How is about the old UTs? it said some HTML tag (<br>, <p> ...) should be replaced by break
Maybe we missed some IT tests for this but we have this case https://ci.open-paas.org/jira/browse/AJ-1